src/sentry/api/endpoints/project_grouping_configs.py

```diff
@@ -21,10 +22,10 @@ def get(self, request, project):
 
         # As long as newstyle:2019-10-29 is the global "latest", allow orgs to upgrade
         # to mobile:2021-02-12 if the appropriate feature flag is on:
-        if latest == "newstyle:2019-10-29" and features.has(
+        if latest == DEFAULT_GROUPING_CONFIG and features.has(
```

Comment:
The reason I hard-coded the strategies here is that the behavior depends on a specific feature flag. As soon as this feature is enabled for everyone, "latest" would now point to `BETA_GROUPING_CONFIG` for everyone. We could of course make it dependent on something more generic like `is_early_adopter()`.

Reply:
ah ok, yeah I still think this is fine as-is for now

we still need to rename the grouping strategy away from mobile to something else so having a central variable is good